src/helpers/DataRequirementHelpers.ts

```diff
@@ -23,8 +23,9 @@ import * as GapsInCareHelpers from '../gaps/GapsReportBuilder';
 import { parseQueryInfo } from '../gaps/QueryFilterParser';
 import * as RetrievesHelper from '../gaps/RetrievesFinder';
 import { uniqBy } from 'lodash';
-import { DateTime, Interval } from 'cql-execution';
+import { DateTime, Interval, Library } from 'cql-execution';
```

Comment:
Looks like `Library` isn't used

```suggestion
import { DateTime, Interval } from 'cql-execution';
```